### ISSUE TITLE
could you please publish to sonatype repo?

### ISSUE THREAD

**Issue by jozic** [mrvisser/sealerate#4]

**Comment by mrvisser** [comment]
Thanks for the suggestion Eugene. I am planning to do this and will look at it soon.

**Comment by jozic** [comment]
sounds great, thanks! let me know if you need help

**Comment by mrvisser** [comment]
FYI, I've published to sonatype and it's in staging (probably not accessible yet). I believe they have a manual process to promote the initial release?

Just to warn you, I've switched the group id to `ca.mrvisser` for this, as I would not be allowed to publish to a `com.pellucid` anymore. No functional changes have been made, so the sonatype version will be available as 0.0.3 as well: `ca.mrvisser:sealerate:0.0.3`

I'll update and resolve this ticket when I've confirmed it's publicly available.
